src/Dns/Dns/help/New-AzDnsZone.md

```diff
@@ -98,6 +105,51 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -ParentZone
```

Comment:
@ShwethaRaghupathi for each of these parameters that have been added in the PR, would you mind adding an example to show users what values should be provided to the parameters and what the corresponding output is?

Reply:
Added